autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -83,6 +84,29 @@ const ConfigureCard: React.FC = () => {
       dispatch(selectSchedule(0));
       setLoading(false);
     });
+
+    // make request to save course cards
+    const courseData = [];
+    for (let i = 0; i < courseCards.numCardsCreated; i++) {
+      const course = courseCards[i];
+      if (course) {
+        const sections = course.sections.filter(({ selected }) => selected).map((section) => (
+          section.section.id
+        ));
```

Comment:
```suggestion
        const sections = course.sections.filter(({ selected }) => selected).map((sectionSel) => (
          sectionSel.section.id
        ));
```

nit: Frankly I always get confused by `SectionSelected`'s, so I would prefer specifying that its not just a normal `Section` object (the `section.section` looks pretty weird too lol)

Reply:
I refactored this so it's no longer relevant